src/codeu/chat/client/commandline/Chat.java

```diff
@@ -47,11 +49,21 @@ public Chat(Context context) {
   // is willing to take another command, the function will return true. If
   // the system wants to exit, the function will return false.
   //
-  public boolean handleCommand(String line) {
+  public boolean handleCommand(String line) throws IOException {
```

Comment:
I backmerged the changes to master into this branch 5 days ago, only difference that wasn't accounted for should be Tammy changing the panel command for version